app/helpers/vacancies_helper.rb

```diff
@@ -72,7 +72,8 @@ def radius_filter_options
 
```

Comment:
What's going on in this commit?
`Update the job role options and subject options for GOVUKDesignSystemFormBuilder`
Your commit messages could afford to be more explicit about the 'why' sometimes - makes it easier to follow along at home / understand changes months and years in the future.

Reply:
GOVUKDesignSystemFormBuilder doesn't implement any `include_blank` type functionality (partly I guess because selects are actively discouraged, so I added a blank subject to the list of options. 

OpenStructs because that's what the documentation recommended, but I've since found another way of solving the same issue, so will readdress this.

Also, noted re commits - will make a conscious effort to be more explicit